src/utils/alignment-D3-support.js

```diff
@@ -53,6 +53,18 @@ export function getShapeValueYearScatter(shape) {
   return parseInt(shape.getAttribute('data-xvalue'), 10);
 }
 
+export function getShapeValueYearBar(shape) {
+  return parseInt(shape.getAttribute('data-date').split('-')[0], 10);
```

Comment:
If you want to leave an inline comment with an example of what the data looks like it will be clearer what the split is doing. If you don't have time, it's not critical.